.github/CONTRIBUTING.md

```diff
@@ -41,6 +41,7 @@ You'll see exactly what is sent, be able to add your comments, and then send it.
 
 ## Communication
 
-* The [#newpipe](irc:irc.freenode.net/newpipe) channel on freenode has the core team and other developers in it. [Click here for webchat](https://webchat.freenode.net/?channels=newpipe)!
-* You can also use a Matrix account to join the Newpipe channel at [#freenode_#newpipe:matrix.org](https://matrix.to/#/#freenode_#newpipe:matrix.org).
+* The #newpipe channel on Libera Chat (`ircs://irc.libera.chat:6697/newpipe`) has the core team and other developers in it. [Click here for webchat](https://web.libera.chat/#newpipe)!
```

Comment:
Hmm, Firefox didn't show me that `#newpipe` is a link to open the IRC channel for some reason. I guess this is actually better than before because of that.

Reply:
I think it's a GitHub issue (or a link which is not allowed in Markdown).

Reply:
Yeah, I think that's better. The `ircs://` link is something only IRC clients can open, right? Then the link should not be clickable.